models/create_inverse_gan_models.py

```diff
@@ -0,0 +1,288 @@
+import logging
```

Comment:
Is the code in this file from you or the paper's authors? If it's your code please add a licence header.

Reply:
this is my own code

Reply:
Great!